src/main/java/uk/ac/sanger/sccp/stan/model/Tissue.java

```diff
@@ -28,11 +28,15 @@ public class Tissue {
     @ManyToOne
     private Hmdmc hmdmc;
     private LocalDate collectionDate;
+    @ManyToOne
+    private SolutionSample solutionSample;
+    private Integer parentId;
 
     public Tissue() {}
 
     public Tissue(Integer id, String externalName, String replicate, SpatialLocation spatialLocation, Donor donor,
-                  Medium medium, Fixative fixative, Hmdmc hmdmc, LocalDate collectionDate) {
+                  Medium medium, Fixative fixative, Hmdmc hmdmc, LocalDate collectionDate,
+                  SolutionSample solutionSample, Integer parentId) {
```

Comment:
Would you please explain what this parentId is for?

Reply:
We are going to have original tissues being processed into blocks, which means tissues are going to be derived from other tissues. The parent id is a foreign key to indicate what tissue it was derived from.